tests_integration/markets/polymarket/test_get_markets.py

```
@@ -49,3 +53,19 @@ def test_many_markets() -> None:
     )
 
     assert len(polymarket_markets) == limit
+
+
+def test_get_positions() -> None:
+    # ToDo - find better address
+    user_address = Web3.to_checksum_address(
+        "0x60073655845c95014a540ae3f7375b15801647a3"
+    )  # top holder in a few markets
+    # Todo - Identify multiple positions per market
+    condition_id = HexBytes(
```

Ruff (0.12.2)

Local variable `condition_id` is assigned to but never used

Remove assignment to unused variable `condition_id`

(F841)